### PR TITLE
Feature/map reorg

### PR DESCRIPTION
**Pull request by egrace479**
- Adds map into Dashboard. Access map in the distribution section through the `Map View` button (return to histogram with `Show Histogram` button. Each has different display options set in `components/divs.py`.
- Graphing functions (for histogram and pie chart) are now in `components/graphs.py` with the map function. DataFrame manipulation and image retrieval functions are in `components/query.py`.
- Set fixed values at top of dashboard file for div styling.
- Removed prototype components from `components` folder, and removed reference to them from `README`.
- Updated preview images (one with histogram and one with map) in `README`.
- Addresses Issues #9 and #13

### CODE REVIEW

**Comment by johnbradley** [comment]
This change adds a new button that switches between the `Map View` and the `Histogram View`.

The top part of the Map View looks like this:
Screen Shot 2023-06-30 at 1 26 26 PM

The top part of the Histogram View looks like this:
Screen Shot 2023-06-30 at 1 26 41 PM

I found it a little surprising how the whole screen switched around when clicking this button.
I'm wondering if there is a better location or perhaps a tab control to make it more obvious what clicking it does.

**Comment by hlapp** [comment]
I agree the UX of switching the view has considerable room for improvement. But I would caution us to keep in mind the context we're still in, namely that of putting together a sufficiently functional prototype, not (yet) that of actual product development.

Perhaps there's an easy enough way to improve the UX on this, but if not, I'd argue allowing a user to submit or specify their own dataset should be the higher order of priority in terms of getting ready for the Image Datapalooza.

File: components/divs.py
Comment (johnbradley) [inline components/divs.py:39]: You may want to remove this commented out code.

File: components/divs.py
Comment (johnbradley) [inline components/divs.py:49]: You may want to remove this commented out code.

File: components/divs.py
Comment (johnbradley) [inline components/divs.py:95]: You may want to remove this commented out code.

**Review by johnbradley — APPROVED** [review]
LGTM. Just a suggestion for a better button name and some there are a few randomly left in commented out bits of code you may want to remove.

**Review by thompsonmj — APPROVED** [review]

**Comment by egrace479** [comment]
> This change adds a new button that switches between the `Map View` and the `Histogram View`.
> 
> The top part of the `Map View` looks like this: Screen Shot 2023-06-30 at 1 26 26 PM
> 
> The top part of the `Histogram View` looks like this: Screen Shot 2023-06-30 at 1 26 41 PM
> 
> I found it a little surprising how the whole screen switched around when clicking this button. I'm wondering if there is a better location or perhaps a tab control to make it more obvious what clicking it does.

I definitely agree that the layout for that quadrant could use some improvement and will set up an issue to brainstorm solutions. I also agree with @hlapp that functionality for Image Datapalooza needs to be prioritized.

**Comment by hlapp** [comment]
BTW @egrace479 given that there've already been two pairs of eagle eyes, I'm not sure there's anything I can add, so please don't wait for my review or approval.

**Pull request merged**
